_spikes-poc/check-allocation-function-poc/table-reset/index.ts

```diff
@@ -0,0 +1,51 @@
+import { AzureFunction, Context, HttpRequest } from '@azure/functions'
+import azureStorageHelper from '../lib/azure-storage-helper'
+import { TableQuery } from 'azure-storage'
+import az = require('azure-storage')
+const tableService = azureStorageHelper.getPromisifiedAzureTableService()
+import { performance } from 'perf_hooks'
+
+const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+  // demonstrate purging of checkAllocation table via delete and recreate
+  const start = performance.now()
+  const tableName = 'checkAllocation'
+  const query = new TableQuery()
+  query.top(200)
+  let done = false
+  let batchNumber = 1
+  while (!done) {
+    const data = await tableService.queryEntitiesAsync(tableName, query, null)

```

Comment:
try/catch here too?

Reply:
only if the catch block does something useful, otherwise its a pointless wrapper. this is POC code anyway, so i wont be productionising it